### PR TITLE
Disallow guests from modifying project

### PR DESCRIPTION
**Pull request by andrerpbts**

### CODE REVIEW

File: app/assets/javascripts/models/story.js [hunk 1]
```diff
@@ -40,9 +40,17 @@ var Story = module.exports = Backbone.Model.extend({
     this.setReadonly();
   },
 
-  setReadonly: function() {
-    if(this.get('state') === 'accepted' && this.get('accepted_at') !== undefined)
-      this.isReadonly = true;
+  setReadonly: function () {
+    var accepted = this.get('state') === 'accepted' && this.get('accepted_at') !== undefined;
+    var isGuest = (
+          this.collection !== undefined &&
```
Comment (andrerpbts) [inline app/assets/javascripts/models/story.js]: Align this like you did in the file bellow.

Comment (brunobrgs) [reply app/assets/javascripts/models/story.js]: Long lines are not problem?

Comment (andrerpbts) [reply app/assets/javascripts/models/story.js]: I mean, like here https://github.com/Codeminer42/cm42-central/pull/243/files#diff-5d8aecce3a76454de0b472c96fa1b6a8R346

Comment (brunobrgs) [reply app/assets/javascripts/models/story.js]: Done

**Pull request merged**